### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by palash23jain**
Screenshot 2022-02-20 at 2 29 32 PM

### CODE REVIEW

File: home/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Batsmen's Paradise</title>
+    <link rel="stylesheet" href="https://bitui.netlify.app/common-css/main.css">
+    <link rel="stylesheet" href="/common-css/main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <div class="navbar-container">
```
Comment (JuvitaSaini09) [inline home/index.html:16]: suggestion:for semantic html ,you can use nav tag

File: home/index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            <img src="/assets/gm.png" alt="">
+        </a>
+    </div>
+    <div class="sachin">
+        <img src="/assets/sachin-bat-willow.jpeg" alt="">
+    </div>
+    <div class="new-arrivals">
+        <div class="card-container card-container-hz-1">
+            <div class="card-img-container-hz-1">
+                <img src="/assets/bats.jpg" alt="" class="card-img">
```
Comment (JuvitaSaini09) [inline home/index.html:44]: alt is empty

File: home/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        </a>
+        <a href="">
+            <img src="/assets/gm.png" alt="">
+        </a>
+        <a href="">
+            <img src="/assets/gm.png" alt="">
+        </a>
+    </div>
+    <div class="sachin">
+        <img src="/assets/sachin-bat-willow.jpeg" alt="">
```
Comment (JuvitaSaini09) [inline home/index.html:39]: empty alt attribute

**Review by saifudinbadani — APPROVED** [review]
Please add a link to connect product listing page, otherwise looks good to merge.

**Comment by JuvitaSaini09** [comment]
add info about image in alt tags in your document.rest of your document looks good to merge

**Review by JuvitaSaini09 — APPROVED** [review]

**Pull request merged**